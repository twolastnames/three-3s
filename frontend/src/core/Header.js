import React from 'react';
import { Link } from 'react-router-dom';

export function Header() {
  return (
    <nav>
      <div className='nav-header'>threAS3</div>
      <ul>
        <li>
          <Link to="/architect">Architect</Link>
        </li>
      </ul>
    </nav>
  );
}
