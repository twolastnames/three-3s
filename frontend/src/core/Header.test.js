import React from 'react';
import { shallow } from 'enzyme';
import { Header } from './Header';

describe('Message', () => {
  it('renders visually', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
