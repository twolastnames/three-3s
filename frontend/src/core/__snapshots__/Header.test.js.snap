// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message renders visually 1`] = `ShallowWrapper {}`;
